content/applications/finance/accounting/fiscal_localizations/localizations/india.rst

```diff
@@ -187,3 +190,152 @@ website.
    .. image:: india/signed-invoice.png
       :align: center
       :alt: verified e-invoice
+
+.. _india/e-waybill:
+
+Indian E-Waybill
+==================
+
+Odoo is compliant with the **Indian Good and Services Tax (GST) E-Waybill system** requirements.
+
+.. important::
+   Indian E-Waybill is available from Odoo 15.0. If needed, :doc:`upgrade
+   </administration/upgrade>` your database.
+
+.. _india/e-waybill-api:
+
+API Registration on your NIC E-Waybill web portal
+--------------------------------------------------
+
+You must register on the **NIC E-Waybill** web portal to create your **API credentials**. You need
+these credentials to :ref:`configure your Odoo Accounting app <india/e-waybill-configuration>`.
+
+#. Login to the NIC E-Waybill web portal on - https://ewaybillgst.gov.in/ by clicking on
+   :guilabel:`Login` and entering your :guilabel:`Username` and :guilabel:`Password`.
+
+   .. image:: india/e-waybill-system-login.png
+      :align: center
+      :alt: E-Waybill login
+
+#. From your dashboard, go to :menuselection:`Registration --> For Gsp`.
+
+   .. image:: india/e-waybill-registration.png
+      :align: center
+      :alt: E-Waybill registration
+
+#. Click Send OTP, you receive an :abbr:`OTP (one-time password)` code to your registered mobile number.
+
+   .. image:: india/e-waybill-gsp-registration.png
+      :align: center
+      :alt: E-Waybill GSP options
+
+#. Enter the OTP code and click on :guilabel:`Verify OTP`.
+
+   .. image:: india/verify-otp.png
+      :align: center
+      :alt: E-Waybill verify otp
+
+#. Check If :guilabel:`Tera Software Limited` is already in List of GSP/ERP Registered.If it is already in list
+   then use that username and password else continue to next steps.
+
+   .. image:: india/e-waybill-gsp-list.png
+      :align: center
+      :alt: E-Waybill added GSP
+
+#. Select :guilabel:`Add/New`, select :guilabel:`Tera Software Limited` as your GSP Name, and create
+   :guilabel:`Username` and :guilabel:`Password` for your API.
+
+   .. image:: india/submit-gsp-api-registration-details.png
+      :align: center
+      :alt: Submit GSP api registration details
+
+#. Click on :guilabel:`Add`.
+
+
+.. _india/e-waybill-configuration:
+
+Configuration on Odoo
+---------------------
+
+To set up the E-Waybill service, go to :menuselection:`Accounting --> Configuration --> Settings -->
+Indian Electronic WayBill`, and enter the :guilabel:`Username` and :guilabel:`Password`.
+
+.. image:: india/e-waybill-setup.png
+   :align: center
+   :alt: E-Waybill setup odoo
+
+.. _india/e-waybill-workflow:
+
+Workflow
+--------
+
+To start E-waybill from Odoo, an invoice/bill must be created with details of E-waybill using the standard
+invoicing/bill flow, that is either from a sales/purchase order or the invoice/bill menu in the Accounting
```

Comment:
I suppose it is the best way, but technically maybe not required...

Reply:
the same thing has been done on the e-invoice also should I remove it from that also?

Reply:
Actually, it is fine as it is, because you describe both flows.  (when you have sale and when not).  My bad.